autoreject/ransac.py

```diff
@@ -20,6 +20,13 @@
 
 
 def _iterate_epochs(ransac, epochs, idxs, verbose):
+
+    # Select random subset
+    if ransac.n_jobs > 1:
```

Comment:
it would be great to have the same code  path and results for any n_jobs.
Can you see how to obtain this? Precompute the seeds or random sample to pass to subprocesses?

Reply:
@LegrandNico did you have time to address this comment? otherwise I'll take over

Reply:
Sorry, I did not have the time + I am not at ease with Joblib. I think this would require to change several parts of the script. Still, I am interested to see how you would do that.